LICENSE.md

```diff
@@ -1,35 +1,37 @@
-                    GNU AFFERO GENERAL PUBLIC LICENSE
-                       Version 3, 19 November 2007
+GNU Affero General Public License
```

Comment:
yes, following on from historical decisions, MIT for libs, and AGPL for apps. I'm considering changing everything to MIT, but that is a different discussion for another time.